frigg/deployments/models.py

```diff
@@ -52,10 +52,16 @@ def is_pending(self):
     @property
     def queue_object(self):
         obj = self.build.queue_object
+
+        if self.image and self.image != "":
```

Comment:
well, we may add more ways to create a PRDeployment-object later, and this will ensure that this is always the behavior we are expecting ?